src/Text.js

```diff
@@ -63,6 +63,12 @@ Text.propTypes = {
   py: numberStringOrArray
 }
 
+Text.h1 = Text.withComponent('h1')
```

Comment:
This looks great! I was suggest that these be rolled into a Heading component per #20 – I think it can make the JSX a little more readable to see where headings are, but open to other thoughts

Reply:
Also we may want to set a default `fontSize` prop that would pick up from `theme.fontSizes` - maybe some sensible default for general headings

Reply:
@jxnblk , cool. I've checked out the details of #20. Thanks for cross-referencing to #48. I've assigned #48 & #20 to myself.

This PR would address the scope of #48. If it looks okay, we can take the PR and resolve #48. I can try out the Heading component in local and tackle #20.

Does that sound like a viable plan? :)

Reply:
Started working on another feature branch named [heading-component](https://github.com/pricelinelabs/design-system/tree/heading-component) to tackle #20 .

Since all tests pass & 1 approval is received, going to take this PR to resolve the scope #48. A new PR from [heading-component](https://github.com/pricelinelabs/design-system/tree/heading-component) will be submitted for #20.